atlasdb-config/src/main/java/com/palantir/atlasdb/factory/Leaders.java

```diff
@@ -175,7 +176,7 @@ public static LocalPaxosServices createInstrumentedLocalServices(
                 acceptors,
                 config.quorumSize(),
                 createExecutorsForService(metricsManager, acceptors, "latest-round-verifier"),
-                true);
+                PaxosConstants.CANCEL_REMAINING_CALLS);
```

Comment:
It looks like you missed changing this for `leaderPinger` defined just below - was that intentional?

Reply:
Nope, that should be changed as well, thanks!